<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoDialog</class>
 <widget class="QDialog" name="UserInfoDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblAvatar">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;Avatar&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblUserName">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;User Name&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblStatus">
         <property name="text">
          <string>&lt;Status&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>151</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPersonal">
      <attribute name="title">
       <string>Personal Information</string>
      </attribute>
      <widget class="QLabel" name="lblFirstNameDesc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblLastNameDesc">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtFirstName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtLastName">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lblAbout">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>281</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>About:</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="txtAbout">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>285</width>
         <height>41</height>
        </rect>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>N/A</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSystem">
      <attribute name="title">
       <string>System Information</string>
      </attribute>
      <widget class="QLabel" name="lblOSNameDesc">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Operating System:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblOSName">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;Operating System&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblLogonName">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;Logon Name&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblIPAddressDesc">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>IP Address:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblVersionDesc">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Messenger Version:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblVersion">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;Messenger Version&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblComputerNameDesc">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Computer Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblIPAddress">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;IP Address&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblLogonNameDesc">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Logon Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblComputerName">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;Computer Name&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>txtFirstName</tabstop>
  <tabstop>txtLastName</tabstop>
  <tabstop>txtAbout</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>UserInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
